schedule

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+from datetime import date, datetime
+from dateutil.relativedelta import relativedelta
+
+def parse_date(value: str) -> date:
+    try:
+        return datetime.strptime(value, '%Y-%m-%d').date()
+    except ValueError as exc:
+        raise SystemExit('Invalid date: ', str(exc)) from exc
+
+
+def week_range(day: date) -> str:
+    weekday = day.weekday()
+    monday = day - relativedelta(days=weekday)
+    friday = day + relativedelta(days=5 - weekday)
+    return f'{monday} - {friday}'
+
+
+def main():
+    dev_start = parse_date(input("Dev start date (YYYY-MM-DD): "))
+
+    ga_date = dev_start + relativedelta(months=3)
```

Comment:
@Ron-Lavi suggested that this should be branching date, not ga_date` since we have 3 months of development time.